build.sbt

```diff
@@ -328,7 +328,7 @@ lazy val interpreter = (project in engine("interpreter")).
     buildInfoPackage := "pl.touk.nussknacker.engine.version",
     buildInfoOptions ++= Seq(BuildInfoOption.ToMap)
   ).
-  dependsOn(util)
+  dependsOn(util, queryableState)
```

Comment:
this adds Flink to interpreter dependencies - let's try to avoid this. Maybe extract trait QueryableStateClient to interpreter (or somewhere...) or detec appropriate ProcessManagerProviders by checking if they implement some additional trait?